File: CPP/Iniciante/1010.cpp
#include <iostream>
#include <iomanip>
 
using namespace std;
 
int main() {
    
    int id, quantity;
    double price;
    
      cin  >> id >> quantity >> price;

    int id_2, quantity_2;
    double price_2;
    
      cin  >> id_2 >> quantity_2 >> price_2;
    
    double answer;
    
      answer = (quantity * price) + (quantity_2 * price_2);
    
      cout << fixed << setprecision(2);
      cout << "VALOR A PAGAR: R$ " << answer << "\n";
   
    return 0;
    
}

File: Python/Iniciante/1010.py
p1 = input().split(" ")

p2 = input().split(" ")

id1, u1, v1 = p1

id2, u2, v2 = p2

total = (int(u1) * float(v1)) + (int(u2) * float(v2))

print(f'VALOR A PAGAR: R$ {total:.2f}')
